packages/kbn-alerts-ui-shared/src/common/hooks/use_search_alerts_query.test.tsx

```diff
@@ -355,150 +362,24 @@ describe('useFetchAlerts', () => {
       },
     });
 
-    dataSearchMock.mockReturnValue(obs$);
-    const { result } = renderHook(() => useFetchAlerts(args));
-    const [_, alerts] = result.current;
+    mockDataPlugin.search.search.mockReturnValue(obs$);
+    const { result } = renderHook(() => useSearchAlertsQuery(params), { wrapper });
 
-    expect(alerts.alerts).toEqual([]);
-  });
-
-  it('resets pagination on refetch correctly', async () => {
```

Comment:
we don't need these tests anymore?

Reply:
I moved the pagination reset logic out of the fetching hook into the AlertsTableState component and so I moved the tests as well, they're now in [`alerts_table_state.test.tsx`](https://github.com/elastic/kibana/pull/186978/files/3f2995e69fd3c0f44507fd51f70c208fd182ee57#diff-7f955eacbafe983d0af231f45ec3e3b5f14d0536bbef74ee454d051985cf9c53R1003) 🙂